### PR TITLE
add security test suite to eden

### PR DESCRIPTION
**Pull request by shjala**
This adds security test suite to eden. As suggested by @uncleDecart, to liberate ourselves from bash and eden script, it is written in go ;)

For now it just check apparmor enablement, more to be added to the suite.

### CODE REVIEW

**Review by uncleDecart — APPROVED** [review]
LGTM, thank you @shjala for this!

cc: @milan-zededa and @giggsoff for suggestions, comments if any

**Comment by shjala** [comment]
I will fix the yetus complains.

File: tests/sec/sec_test.go [hunk 1]
```diff
@@ -0,0 +1,110 @@
+package sec
```
Comment (uncleDecart) [inline tests/sec/sec_test.go:1]: That's a reasonable complaint

**Review by milan-zededa — COMMENTED** [review]
Overall this is OK, although there is quite a lot of boiler-plate out of which only one line is the actual test. Currently, I would not say that writing eden tests in golang is easier than with escript, but hopefully over time we can improve the test framework and make it more user friendly.

**Pull request merged**